toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -211,13 +213,72 @@ Preferences::LevelFormat PreferencesPopup::FormatProperties::levelFormat()
 //    PreferencesPopup  implementation
 //**********************************************************************************
 
+void PreferencesPopup::onPixelsOnlyChanged(int index) {
+  bool enabled = index == Qt::Checked;
+  if (enabled) {
+    m_pref->setDefLevelDpi(53.33333);
+    m_pref->setPixelsOnly(true);
+    TCamera *camera;
+    camera =
+        TApp::instance()->getCurrentScene()->getScene()->getCurrentCamera();
+    TDimension camRes = camera->getRes();
+    TDimensionD camSize;
+    camSize.lx = camRes.lx / 53.33333;
+    camSize.ly = camRes.ly / 53.33333;
+    camera->setSize(camSize);
```

Comment:
Here you are changing the size of the current camera with dpi value 53.33333.
I think it would be better to do the same thing for the cleanup camera of current cleanup settings as well.
It is stored in CleanupSettingsModel::instance()->getCurrentParameters()->m_camera .